src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -169,19 +169,22 @@ components:
       properties:
         deliveryClass:
           $ref: '#/components/schemas/DeliveryClass'
-        deliveryMethod:
-          $ref: '#/components/schemas/DeliveryMethod'
+        addressType:
+          $ref: '#/components/schemas/DeliveryAddressType'
         addressee:
           type: string
           maxLength: 255
           description: The addressee to be printed in addition to the delivery address
         address:
           $ref: '#/components/schemas/Address'
+        addressFormat:
+          $ref: '#/components/schemas/AddressFormat'
       required:
-        - deliveryOption
```

Comment:
I think `deliveryOption` was a very early version of this! Should have been `deliveryClass`

Reply:
Without versioning the SQS messages and having different listeners for the different models we are in a position where messages will fail to be processed.

Deployment Issue
Release Print first to support version `1.9.0` of spec - will fail to process any messages from spec `1.8.0`
Release VCA first to send  version `1.9.0` of spec - print will fail to process as missing required properties from spec `1.8.0`

At least in this case messages end up on DLQ and can be Redrived.